### PR TITLE
Added an implementation of the InsertStatusValueRequest method

### PR DESCRIPTION
**Pull request by bwmodular**
Added an implementation of the InsertStatusValueRequest method as an InsertStatusValueRequestExecutor.

Along with the executor and associated tests, I've also had to add a new dictionary to the faked context metadata to hold the StatusAttributeMetadata values. Hope that was the right thing to do - all tests running OK, so think it's OK.

### CODE REVIEW

File: FakeXrmEasy.Shared/FakeXrmEasy.Shared.projitems [hunk 1]
```diff
@@ -85,4 +86,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)XrmOrderByAttributeComparer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)XrmRealContext.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="$(MSBuildThisFileDirectory)FakeMessageExecutors\NewFolder\" />
```
Comment (eNeRGy164) [inline FakeXrmEasy.Shared/FakeXrmEasy.Shared.projitems]: I guess this is not intended :)

Comment (bwmodular) [reply FakeXrmEasy.Shared/FakeXrmEasy.Shared.projitems]: Oops! New to Git - how do I deal with this? Delete it locally and then....?

Comment (eNeRGy164) [reply FakeXrmEasy.Shared/FakeXrmEasy.Shared.projitems]: Just remove it, commit + push

Comment (bwmodular) [reply FakeXrmEasy.Shared/FakeXrmEasy.Shared.projitems]: Thanks - sorted now...

Comment (jordimontana82) [reply FakeXrmEasy.Shared/FakeXrmEasy.Shared.projitems]: Cheers guys

**Comment by jordimontana82** [comment]
Hi Ben,

This is in v1.39.4 now.

**Pull request merged**